demo/image_impressionism/build.gradle

```diff
@@ -9,6 +9,8 @@ repositories {
 }
 
 dependencies {
+    compile 'org.scala-lang:scala-library:2.10.4'
```

Comment:
We are compiling against Spark 2.10, so we shouldn't use 2.11, it will be inconsistent.